docs/index.md

```diff
@@ -40,11 +40,11 @@ resource "wallarm_rule_vpatch" "vpatch" {
-* `api_secret` - (Required) Your Wallarm user secret. Note that the most operations with Wallarm API are allowed only for the users with the **Administrator** role. To get UUID, please the [instruction](https://docs.wallarm.com/admin-en/api-en/#your-own-client). This can also be specified with the `WALLARM_API_SECRET` shell environment variable.
-* `api_host` - (Optional) Wallarm API URL. Can be: `https://us1.api.wallarm.com` for the [US cloud](https://docs.wallarm.com/quickstart-en/how-wallarm-works/qs-intro-en/#us-cloud), `https://api.wallarm.com` for the [EU cloud](https://docs.wallarm.com/quickstart-en/how-wallarm-works/qs-intro-en/#eu-cloud). This can also be specified with the `WALLARM_API_HOST` shell environment variable. Default: `https://api.wallarm.com`.
-* `client_id` - (Optional) ID of the client to apply the rules to. The value is required for multi-tenant scenarios. This can also be specified with the `WALLARM_API_CLIENT_ID` shell environment variable. Default: client ID of the authenticated user defined by UUID and SECRET.
-* `retries` - (Optional) Maximum number of retries to perform when an API request fails. Default: 3. This can also be specified with the `WALLARM_API_RETRIES` shell environment variable.
-* `min_backoff` - (Optional) Minimum backoff period in seconds after failed API calls. Default: 1. This can also be specified with the `WALLARM_API_MIN_BACKOFF` shell environment variable.
-* `max_backoff` - (Optional) Maximum backoff period in seconds after failed API calls Default: 30. This can also be specified with the `WALLARM_API_MAX_BACKOFF` shell environment variable.
-* `api_client_logging` - (Optional) Whether to print logs from the API client (using the default log library logger). Default: false. This can also be specified with the `WALLARM_API_CLIENT_LOGGING` shell environment variable.
+* `api_uuid` - (**required**) Your Wallarm user UUID. Note that the most operations with Wallarm API are allowed only for the users with the **Administrator** role. To get UUID, please the [instruction](https://docs.wallarm.com/admin-en/api-en/#your-own-client). This can also be specified with the `WALLARM_API_UUID` shell environment variable.
+* `api_secret` - (**required**) Your Wallarm user secret. Note that the most operations with Wallarm API are allowed only for the users with the **Administrator** role. To get UUID, please the [instruction](https://docs.wallarm.com/admin-en/api-en/#your-own-client). This can also be specified with the `WALLARM_API_SECRET` shell environment variable.
+* `api_host` - (optional) Wallarm API URL. Can be: `https://us1.api.wallarm.com` for the [US cloud](https://docs.wallarm.com/quickstart-en/how-wallarm-works/qs-intro-en/#us-cloud), `https://api.wallarm.com` for the [EU cloud](https://docs.wallarm.com/quickstart-en/how-wallarm-works/qs-intro-en/#eu-cloud). This can also be specified with the `WALLARM_API_HOST` shell environment variable. Default: `https://api.wallarm.com`.
```

Comment:
capitalize Cloud please

Reply:
links are super old

Reply:
Checked links, they seem ok, what is wrong with them?

Reply:
they redirect you to new ones, it is better to replace the old links with new ones